editor/src/clj/editor/scene.clj

```diff
@@ -1236,10 +1236,10 @@
           (.setImage image-view new-image))))))
 
 (defn- nudge! [scene-node-ids ^double dx ^double dy ^double dz]
-  (g/transact
-    (for [node-id scene-node-ids
-          :let [[^double x ^double y ^double z] (g/node-value node-id :position)]]
-      (g/set-property node-id :position [(+ x dx) (+ y dy) (+ z dz)]))))
+  (g/with-auto-evaluation-context evaluation-context
+    (g/transact
```

Comment:
```suggestion
  (g/transact
    (g/with-auto-evaluation-context evaluation-context
```
I would swap the call order so that `g/transact` surrounds the `g/with-auto-evaluation-context` block. 

This is because `g/with-auto-evaluation-context` will transfer any changes that have been made to the local cache inside the `evaluation-context` to the system cache so they're visible to everyone at the end of its scope. But `g/transact` may also cause entries to be evicted from the cache due to its changes.

We have code to deal with conflicts like this in `internal.system/update-cache-from-evaluation-context`, but for efficiency, we want to avoid situations where an `g/with-auto-evaluation-context` block surrounds code that modifies the graph state.